.github/workflows/validate.yml

```diff
@@ -103,35 +108,29 @@ jobs:
     with:
       package: broker
       docker-services: cassandra init-keyspace parity-sidechain-node0 graph-deploy-streamregistry-subgraph
+      run-entry-point: true
       command: npm run test-integration
   client-browser:
     needs: build
     uses: ./.github/workflows/test-setup.yml
     with:
       package: client
       docker-services: init-keyspace parity-sidechain-node0 graph-deploy-streamregistry-subgraph
-      run-brokers-and-trackers: true
+      run-brokers: true
       command: npm run test-browser
   cli-tools:
     needs: build
     uses: ./.github/workflows/test-setup.yml
     with:
       package: cli-tools
-      docker-services: parity-node0 graph-deploy-streamregistry-subgraph broker-node-storage-1
+      docker-services: cassandra init-keyspace parity-sidechain-node0 graph-deploy-streamregistry-subgraph
```

Comment:
Why these dependencies have changed?

Reply:
IIRC, the docker dependencies did not run correctly with the original setup.